src/cascadia/TerminalSettingsEditor/ColorSchemesPageViewModel.cpp

```diff
@@ -31,6 +31,12 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     {
         _settings = settings;
 
+        // Exit rename mode if we're in it
+        if (_InRenameMode)
+        {
+            InRenameMode(false);
+        }

```

Comment:
```suggestion
            InRenameMode(false);
```
I think the setter already checks if the value was different? I could be wrong though.

Reply:
oops you're right! thanks